omnibus/files/server-ctl-cookbooks/infra-server/templates/default/sv-redis_lb-run.erb

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 exec 2>&1
-exec /opt/<%= ChefUtils::Dist::Org::LEGACY_CONF_DIR %>/embedded/bin/veil-env-helper -f /etc/<%= ChefUtils::Dist::Org::LEGACY_CONF_DIR %>/private-<%= ChefUtils::Dist::Infra::SHORT %>-secrets.json -s REDIS_PASSWORD=redis_lb.password -- chpst -P -o 131071 -u <%= node["private_chef"]["user"]["username"] %> -U <%= node["private_chef"]["user"]["username"] %> -o 100000 env HOME="<%= node["private_chef"]["redis_lb"]["dir"] %>" /opt/<%= ChefUtils::Dist::Org::LEGACY_CONF_DIR %>/embedded/bin/redis-server <%= File.join(node["private_chef"]["redis_lb"]["dir"], "etc", "redis.conf") %>
+exec /opt/<%= ChefUtils::Dist::Org::LEGACY_CONF_DIR %>/embedded/bin/veil-env-helper -f /etc/<%= ChefUtils::Dist::Org::LEGACY_CONF_DIR %>/private-<%= ChefUtils::Dist::Infra::SHORT %>-secrets.json -s REDIS_PASSWORD=redis_lb.password -- chpst -P -o 131071 -u <%= node["private_chef"]["user"]["username"] %> -U <%= node["private_chef"]["user"]["username"] %> -o 100000 env HOME="<%= node["private_chef"]["redis_lb"]["dir"] %>" /opt/<%= ChefUtils::Dist::Org::LEGACY_CONF_DIR %>/embedded/bin/redis-server <%= File.join(node["private_chef"]["redis_lb"]["dir"], "etc", "redis.conf") %> --requirepass "${REDIS_PASSWORD}"
```

Comment:
Was the REDIS_PASSWORD passed to this script previously but never used?

Reply:
In redis 5, The password is getting set implicitly but not happening in redis 6. So using extra argument `requirepass` while start the redis 6.
There are some major changes in Redis auth module in version 6. From redis 6 we need to explicitly set password for the default user if we want to enable external access due to ACL's.
For backward compatibility they enabled some user 'default' which should be passed when ever we use Redis.new and password should also be set if we want to access externally.
Reference - https://redis.io/docs/manual/security/acl/